packages/flutter_tools/lib/src/commands/create.dart

```diff
@@ -502,10 +507,42 @@ You've created a plugin project that doesn't yet support any platforms.
 ''');
 }
 
-void _printAddPlatformMessage(String pluginPath) {
+void _printPluginAddPlatformMessage(String pluginPath) {
   globals.printStatus('''
 To add platforms, run `flutter create -t plugin --platforms <platforms> .` under $pluginPath.
 For more information, see https://flutter.dev/go/plugin-platforms.
 
 ''');
 }
+
+// shows warning if user requested a platform and if it's not enabled
+void _printWarningToEnablePlatforms(
+  List<String> requestedPlatforms,
+  [bool isApp = false]
+) {
+  final List<String> _showWarning = <String>[
+  if (requestedPlatforms.contains('web') && !featureFlags.isWebEnabled)
+    'web',
+  if (requestedPlatforms.contains('macos') && !featureFlags.isMacOSEnabled)
+    'macos',
+  if (requestedPlatforms.contains('windows') && !featureFlags.isWindowsEnabled)
+    'windows',
+  if (requestedPlatforms.contains('linux') && !featureFlags.isLinuxEnabled)
+    'linux',
+  ];
```

Comment:
I think we should have Android and iOS as well since they can be enabled/disabled as well!

Reply:
Oh, I learnt a new thing today. Will do need full in some time. Thanks